net/bluetooth/bluetooth_callback.c

```diff
@@ -2,33 +2,20 @@
  * net/bluetooth/bluetooth_callback.c
  * Forward events to waiting PF_BLUETOOTH sockets.
```

Comment:
This comment should be moved out of the license header.

Reply:
was fixed in the right way and the comment is moved after the license